tests/conftest.py

```diff
@@ -76,4 +81,143 @@ def mock_blob():
 
 @pytest.fixture
 def mock_bq_client():
-    return MagicMock(spec=BQClient)
\ No newline at end of file
+    return MagicMock(spec=BQClient)
+
+
+@pytest.fixture
+def transform(mock_gcs_client, mock_bq_client):
+    return Transform(
+        storage_client=mock_gcs_client,
+        bq_client=mock_bq_client,
+        bucket_name="project-cryptocurrency"
+    )
+
+
+@pytest.fixture
+def date_dim_rows():
+    today = datetime.today()
```

Comment:
do we need to "freeze" this for testing?

Reply:
ok maybe I need to  look  at the date_dim_row implementation ...

yep in that case the recommended lib is freezegun, related article: https://pybit.es/articles/guest-freezegun

Reply:
Ok, cool. I'll work on adding it.